components/connect/connect.c

```diff
@@ -102,7 +102,7 @@ esp_netif_t * wifi_init_softap(void)
     strncpy((char *) wifi_ap_config.ap.ssid, ssid_with_mac, sizeof(wifi_ap_config.ap.ssid));
     wifi_ap_config.ap.ssid_len = strlen(ssid_with_mac);
     wifi_ap_config.ap.channel = 1;
-    wifi_ap_config.ap.max_connection = 30;
+    wifi_ap_config.ap.max_connection = 10;
```

Comment:
Something much lower should be fine right e.g. 3?

Reply:
I don't know exactly what counts as a connection, is each connecting client a connection? Or devices polling networks, so no idea how quickly one would run into this limit, so I put it just low enough to get rid of the warning.